index.html

```diff
@@ -8,49 +9,253 @@
+                have a project you need coded, don’t<br>
+                hestiate to contact me.
+            </p>
+        </div>
+        <div class="social-media">
+            <h3 class="contact">LET'S CONNECT </h3>
+            <ul class="social-link">
+                <li class="tweet"><a href="#">
                         <i class="fa fa-twitter fa-2x" aria-hidden="true"></i>
-                        </a>
-                    </li>
-                    <li><a href="#">
+                    </a>
+                </li>
+                <li><a href="#">
                         <i class="fa fa-linkedin fa-2x" aria-hidden="true"></i>
-                        </a>
-                    </li>
-                    <li><a href="#">
+                    </a>
+                </li>
+                <li><a href="#">
                         <i class="fa fa-medium fa-2x" aria-hidden="true"></i>
-                        </a>
-                    </li>
-                    <li><a href="#">
+                    </a>
+                </li>
+                <li><a href="#">
                         <i class="fa fa-github fa-2x" aria-hidden="true"></i>
-                        </a>
-                    <li><a href="#">
+                    </a>
+                <li><a href="#">
                         <i class="fa fa-angellist fa-2x" aria-hidden="true"></i>
-                        </a>
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <button type="button">Get My Resume</button>
+        </div>
+
+        <div class="skills">
+
+            <div class="Language">
+                <h3>Languages</h3>
+                <i class="fa fa-angle-down" aria-hidden="true"></i>
+            </div>
+
+            <div>
+                <ul class="icon-languages">
+                    <li>
+
+                        <img src="images/E0.png" alt="" />JavaScript
+
+
+                    </li>
+
+                    <li>
+
+                        <img src="images/E1.png" alt="" /> HTML
+
+                    </li>
+
+                    <li>
+
+                        <img src="images/E3.png" alt="" />CSS
+
                     </li>
                 </ul>
+                <div>
+                    <ul class="frame">
+                        <li>
+                            <h4>Framework</h4>
+                        </li>
+                        <li><i class="fa fa-angle-right" aria-hidden="true"></i></li>
+                    </ul>
+                </div>
+                <hr class="line">
+                <div>
+                    <ul class="frame">
+                        <li>
+                            <h4>Skills</h4>
+                        </li>
+                        <li><i class="fa fa-angle-right" aria-hidden="true"></i></li>
+                    </ul>
+                </div>
+                <hr class="line">
             </div>
-    
-        </section>
-    
+
+
+        </div>
```

Comment:
[OPTIONAL] Good job here 👍🏽 but for semantics reasons the technology categories ("Languages", "Frameworks" and "Skills") would be better in a `ul`, each of them in a `li` 🤓